examples/room-manager/room_service.py

```diff
@@ -0,0 +1,166 @@
+from argparse import Namespace
+from dataclasses import dataclass
+from logging import Logger
+from typing import List
+
+import betterproto
+
+from fishjam import FishjamClient, PeerOptions, Room, RoomOptions
+from fishjam.events import ServerMessagePeerCrashed as PeerCrashed
+from fishjam.events import ServerMessagePeerDeleted as PeerDeleted
+from fishjam.events import ServerMessageRoomCrashed as RoomCrashed
+from fishjam.events import ServerMessageRoomDeleted as RoomDeleted
+
+
+@dataclass
+class Info:
```

Comment:
Maybe `Resource` or `Entity`
```suggestion
class Resource:
```